### ISSUE TITLE
nothing is blocked anymore

### ISSUE THREAD

**Issue by tsuyoshi2** [proginosko/LeechBlockNG#263]
I'm running Debian. I have tried to use all versions of firefox and firefox-esr currently available in the Debian repositories (stable and unstable) with no luck. Cleared all data, purged Firefox and reinstalled (which necessarily required reinstalling Leechblock), removed ~/.mozilla - none of which worked.

To simplify things, I have reduced my block list to the single line:

*twitter.com

#262 looks pretty similar, although I have not tried to export my options at all. Anyway, I tried to do the same things to figure out the problem.

When I try the diagnostics page, I get:

====== Block Set 1
BLOCK: https://twitter.com

When I enable diagnostic mode and then open the browser console, I get:

Content Security Policy: Ignoring “'unsafe-inline'” within script-src or style-src: nonce-source or hash-source specified 3
Content Security Policy: Ignoring “'unsafe-inline'” within script-src or style-src: nonce-source or hash-source specified 11
Content Security Policy: Ignoring “'unsafe-inline'” within script-src or style-src: nonce-source or hash-source specified 2
Content Security Policy: Ignoring “'unsafe-inline'” within script-src or style-src: nonce-source or hash-source specified 

But that appears to be the same thing I get with or without Leechblock.

I'm willing to try anything... but I guess in the meantime I will be going back to the /etc/hosts method of blocking things.

**Comment by proginosko** [comment]
Can you export your options to a file and email it to me (leechblockng@proginosko.com)?

**Comment by tsuyoshi2** [comment]
OK, I have sent the options.

In the meantime I have noticed that I have another computer where Leechblock is still working. It is running an older version of Firefox (103.0). I tried copying the .deb from that computer to this one where it doesn't work and downgrading. The downgrade went through without a hitch (I am using the old version to write this comment), but still, Leechblock isn't blocking. Honestly, I don't understand how this is even possible... looking at the dependencies, it's mostly the usual GTK libraries. I checked and both are running the same version of Leechblock (1.5.1).
